includes/admin.form.inc

```diff
@@ -65,6 +65,16 @@ function islandora_internet_archive_bookreader_admin_settings_form(array $form,
       '#title' => t('Force the Internet Archive Bookreader into full screen for mobile users?'),
       '#default_value' => variable_get('islandora_internet_archive_bookreader_mobile_full_screen', FALSE),
     ),
+    // XXX: This variable setting exists because Djatoka won't return JP2s
+    // below a filesize that appears to be completely undocumented. Should do
+    // some testing to determine the exact number, but setting at 10kb for now.
```

Comment:
I think this had been tracked down, once upon a time... Pretty sure 4096 bytes was the magick number... Whether that's actually "dynamically" tied to the (memory) page size, or just a nice round power of two in code somewhere... Shrug? For narrowing in later?... Whatever...